api/api.proto

```diff
@@ -805,7 +820,7 @@ service OpenStorageVolume {
   rpc Create(SdkVolumeCreateRequest)
     returns (SdkVolumeCreateResponse) {
       option(google.api.http) = {
-        post: "/v1/volume"
+        post: "/v1/volumes"
```

Comment:
why **volumes** -> volume create/delete etc will always be for single operation right?

Reply:
It was decided in the review to make them all plural